### ISSUE TITLE
Incorrect/incomplete entry for C1694 "stammer"

### ISSUE THREAD

**Issue by amir-zeldes** [KELLIA/dictionary#225]
Entry C1694 looks like the word ⲗⲁⲥ by itself can mean stammering, and it is listed in Crum under the headword for ⲗⲁⲥ. However, this sense only occurs when the word ⲗⲁⲥ is modified by ϭⲁⲁϫⲉ or similar, "cut of tongue", as Crum also lists. This also appears to be the only way that ⲗⲁⲥ appears in Scriptorium translations that contain "stammer", i.e. together with ϭⲁⲁϫⲉ. 

Can this entry be updated to reflect that the actual form of C1694 is not just ⲗⲁⲥ, but more specifically ⲗⲁⲥ ⲉⲧϭⲁⲁϫⲉ? The way it is right now is misleading, because it gives the impression that ⲗⲁⲥ by itself can mean "stammer":

https://coptic-dictionary.org/entry.cgi?tla=C1694

**Comment by simondschweitzer** [comment]
Thanks! In the next version, the entry will be:
```
        <entry type="compound" xml:id="C1694">
          <form type="lemma" xml:id="CF4903">
            <usg type="geo">S</usg>
            <orth>ⲗⲁⲥ ⲉⲧϭⲁⲁϫⲉ</orth>
          </form>
          <form xml:id="CF4904">
            <usg type="geo">S</usg>
            <orth>ⲗⲁⲥ ⲉⲧϭⲁⲁϫⲉ</orth>
          </form>
          <gramGrp>
            <pos>Adj.</pos>
          </gramGrp>
          <sense xml:id="CS1960">
            <cit type="translation">
              <quote xml:lang="de">stotternd</quote>
              <quote xml:lang="en">stammering</quote>
              <quote xml:lang="fr">qui bégaie</quote>
              <bibl>CD 144b-145a</bibl>
            </cit>
          </sense>
        </entry>
```
Could you please add https://annis.copticscriptorium.org/annis/?id=22dff679-33cd-402b-aa2f-af91b2356ec5 and https://annis.copticscriptorium.org/annis/?id=a9e30500-b400-48af-b6be-4e7734336566 as examples in CDO?

**Comment by amir-zeldes** [comment]
Yes, those look great - thanks! It will take me a few days to get to this but it's on my list.
